src/style.css

```diff
@@ -0,0 +1,132 @@
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  width: 200px;
+  height: 408px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  display: block;
+  width: 160px;
+  height: 134px;
+  margin: 0 3px 40px;
+}
+
+.card__about {
+  width: 166px;
+  height: 36px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  font-style: normal;
+  color: #060b35;
+
+  margin: 0 0 4px;
+}
+
+.card__code {
+  width: 98px;
+  height: 14px;
+
+  font-weight: 400;
+  font-size: 10px;
+
+  color: #616070;
+
+  margin: 0 0 16px;
+}
+
+.card__rating {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 24px;
+}
+
+.stars {
```

Comment:
Create separate files for `stars` and `card` BEM blocks and move their styles there
![image](https://user-images.githubusercontent.com/96409563/228442144-a6f44978-7d75-41c9-8d07-5271ea1f6cf0.png)